### PR TITLE
Update xterm w/ npm auto-update

### PR DESCRIPTION
**Pull request by mruoss**
Adapt autoupdate info to comply with new folder structure.

### CODE REVIEW

File: packages/x/xterm.json [hunk 1]
```diff
@@ -1,37 +1,34 @@
...
       }
-    ],
-    "ignoreVersions": [
-      "3.*-beta*",
-      "3.14.0"
     ]
   },
+  "optimization": {
+    "js": false
+  },
```
Comment (MattIPv4) [inline packages/x/xterm.json:28]: Is there a reason for disabling JS optimization as part of this PR? Was there an issue with the minification in previous versions?

File: packages/x/xterm.json [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
-    "target": "xterm",
+    "target": "@xterm/xterm",
     "fileMap": [
       {
-        "basePath": "dist",
-        "files": [
-          "**/*"
-        ]
+        "basePath": "lib",
+        "files": ["*.js", "*.map"]
```
Comment (MattIPv4) [inline packages/x/xterm.json:18]:
```suggestion
        "files": [
          "*.js?(.map)"
        ]
```
We can look for sourcemaps of JS files specifically (and keeping each array item on a new line)

File: packages/x/xterm.json [hunk 3]
```diff
@@ -1,37 +1,34 @@
...
+      },
+      {
+        "basePath": "css",
+        "files": ["*.css"]
```
Comment (MattIPv4) [inline packages/x/xterm.json:22]:
```suggestion
        "files": [
          "*.css"
        ]
```
Keeping each array item on a new line

File: packages/x/xterm.json [hunk 4]
```diff
@@ -1,37 +1,34 @@
...
     "type": "git",
-    "url": "git+https://github.com/sourcelair/xterm.js"
+    "url": "git+https://github.com/xtermjs/xterm.js"
   },
-  "keywords": [
-    "xterm",
-    "terminal",
-    "browser-tty"
-  ],
+  "keywords": ["xterm", "terminal", "browser-tty"],
```
Comment (MattIPv4) [inline packages/x/xterm.json:10]:
```suggestion
  "keywords": [
    "xterm",
    "terminal",
    "browser-tty"
  ],
```
Keeping each array item on a new line

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/x/xterm.json (hunk 1 above)
Comment (mruoss) [reply packages/x/xterm.json:28]: The source JS is already minified. I guess the source map doesn't make much sense if we further minify the code?

Comment (mruoss) [reply packages/x/xterm.json:36]: This seems to prevent you to merge the PR, right? Anything I can do to help?

Comment (MattIPv4) [reply packages/x/xterm.json:36]: Disabling it is fine, just wanted to check it was intentional :)

File: packages/x/xterm.json [hunk 5]
```diff
@@ -1,37 +1,42 @@
...
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/sourcelair/xterm.js"
+    "url": "git+https://github.com/xtermjs/xterm.js"
   },
   "keywords": [
-    "xterm",
-    "terminal",
+    "xterm", 
+    "terminal",
```
Comment (MattIPv4) [inline packages/x/xterm.json:12]:
```suggestion
    "xterm",
    "terminal",
```

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**